### ISSUE TITLE
Opentofu Runner fails to start on appliances

### ISSUE THREAD

**Issue by agrare** [ManageIQ/manageiq-providers-embedded_terraform#31]
TLDR:
- [x] Missing tls.key, tls.crt https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/34
- [x] Missing env vars for database/memcached connection https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/32
- [x] Connection Refused for ::1:5432 and 127.0.0.1:5432 without --net=host https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/40
- [x] Expose port 6000 to hit the internal API https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/39
- [x] `[ERROR] migrate-schema - error: could not open extension control file` https://github.com/ManageIQ/manageiq-rpm_build/pull/461
- [x] Add TERRAFORM_RUNNER_URL env var to /etc/default/*.properties https://github.com/ManageIQ/manageiq-appliance/pull/385

Encountering a number of issues with the opentofu-runner on appliances running with podman

```
[2024-05-23T18:52:22.876] [INFO] index - Enable Https
node:internal/fs/utils:356
    throw err;
    ^
Error: ENOENT: no such file or directory, open '/opt/app-root/src/config/cert/tls.key'
    at Object.openSync (node:fs:596:3)
    at readFileSync (node:fs:464:35)
    at getKey (/opt/app-root/src/dist/utils/ssl_utils.js:8:38)
    at Object.<anonymous> (/opt/app-root/src/dist/index.js:47:50)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
    at node:internal/main/run_main_module:28:49 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/opt/app-root/src/config/cert/tls.key'
}
Node.js v18.20.2
```

Workaround:
```
# mkdir -p /var/lib/manageiq/opentofu-runner/certs
# cd /var/lib/manageiq/opentofu-runner/certs/
# openssl req -x509 -newkey rsa -days 1095 -keyout tls.key -out tls.crt -subj "/CN=server" -nodes -batch
..+......+...+......+......+..........+..+...+...............+....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*..+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*..........+.......+...........+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
..+....+...+..+......+...+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.+...+.........+...+..........+..............+......+....+.....+.........+.+...+...+.....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*...+....+..+.+..+...+....+..............+.......+...+......+...+.....+..........+........+...+.+.....+....+...........+....+.....+.+...........+....+......+...+..+.........+...+.+.........+......+..............+..........+..+.+........+.+.........+.....+.+......+.....+...+.+......+...+..+...+....+...+..+...+.......+............+...+.....+.............+....................+.........+......+...+.+.....+....+..+...+....+..+...+..........+......+......+......+.....+...+.......+...+.....+.......+..+...+.........+.+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-----
# chown -R manageiq\: /var/lib/manageiq/opentofu-runner
# chmod -R 0777 /var/lib/manageiq/opentofu-runner/certs
# chmod 0666 /var/lib/manageiq/opentofu-runner/certs/*
```

Add `--volume=/var/lib/manageiq/opentofu-runner/certs:/opt/app-root/src/config/cert:z` to `ExecStart` in `opentofu-runner.service`

Next:
`INFO] dbs-helper - @@@@@@@@@@@@@@@@@@ PG CONFIG: {"host":"localhost","port":"5432","user":"null","password":"smartvm","ssl":{"rejectUnauthorized":false},"database":"postgres"}`

Database env vars aren't being passed correctly,
Fix:
Add `--env=DATABASE_HOSTNAME=${DATABASE_HOSTNAME} --env=DATABASE_NAME=${DATABASE_NAME} --env=DATABASE_USERNAME=${DATABASE_USERNAME} --env=MEMCACHED_SERVER=${MEMCACHED_SERVER}` to `ExecStart` in `opentofu-runner.service`

Then
```
Connection fails: Error: connect ECONNREFUSED 127.0.0.1:5432
It will be retried for the next request.
node:events:495
      throw er; // Unhandled 'error' event
      ^

Error: connect ECONNREFUSED 127.0.0.1:5432
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
Emitted 'error' event on DataSource instance at:
    at DataSource.postInit (/opt/app-root/src/node_modules/loopback/node_modules/loopback-datasource-juggler/lib/datasour>
    at PendingItem.callback (/opt/app-root/src/node_modules/loopback-connector-postgresql/lib/postgresql.js:130:17)
    at Client._connectionCallback (/opt/app-root/src/node_modules/pg-pool/index.js:254:23)
    at Client._handleErrorWhileConnecting (/opt/app-root/src/node_modules/pg/lib/client.js:327:19)
    at Client._handleErrorEvent (/opt/app-root/src/node_modules/pg/lib/client.js:337:19)
    at Connection.emit (node:events:517:28)
    at Socket.reportStreamError (/opt/app-root/src/node_modules/pg/lib/connection.js:58:12)
    at Socket.emit (node:events:517:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -111,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 5432
}

Node.js v18.20.2
```

Workaround: Add `--net=host` to `ExecStart` in `opentofu-runner.service` (NOTE this is bad)

Then:
```
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'extension.c',
  line: '496',
  routine: 'parse_extension_control_file'
}
[2024-05-23T19:04:42.715] [ERROR] migrate-schema - error: could not open extension control file "/usr/share/pgsql/extensi>
    at Parser.parseErrorMessage (/opt/app-root/src/node_modules/pg-protocol/dist/parser.js:287:98)
    at Parser.handlePacket (/opt/app-root/src/node_modules/pg-protocol/dist/parser.js:126:29)
    at Parser.parse (/opt/app-root/src/node_modules/pg-protocol/dist/parser.js:39:38)
    at TLSSocket.<anonymous> (/opt/app-root/src/node_modules/pg-protocol/dist/index.js:11:42)
    at TLSSocket.emit (node:events:517:28)
    at addChunk (node:internal/streams/readable:368:12)
    at readableAddChunk (node:internal/streams/readable:341:9)
    at Readable.push (node:internal/streams/readable:278:10)
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 187,
  severity: 'ERROR',
  code: '58P01',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'extension.c',
  line: '496',
  routine: 'parse_extension_control_file'
}
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

error: could not open extension control file "/usr/share/pgsql/extension/uuid-ossp.control": No such file or directory
    at Parser.parseErrorMessage (/opt/app-root/src/node_modules/pg-protocol/dist/parser.js:287:98)
    at Parser.handlePacket (/opt/app-root/src/node_modules/pg-protocol/dist/parser.js:126:29)
    at Parser.parse (/opt/app-root/src/node_modules/pg-protocol/dist/parser.js:39:38)
    at TLSSocket.<anonymous> (/opt/app-root/src/node_modules/pg-protocol/dist/index.js:11:42)
    at TLSSocket.emit (node:events:517:28)
    at addChunk (node:internal/streams/readable:368:12)
    at readableAddChunk (node:internal/streams/readable:341:9)
    at Readable.push (node:internal/streams/readable:278:10)
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 187,
  severity: 'ERROR',
  code: '58P01',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'extension.c',
  line: '496',
  routine: 'parse_extension_control_file'
}

Node.js v18.20.2
```

**Comment by agrare** [comment]
Opened https://github.com/ManageIQ/manageiq-providers-embedded_terraform/issues/44 to track the issue
